Packages/Newtonsoft.Json-for-Unity.Converters/Editor/UnityConvertersConfigEditor.cs

```diff
@@ -0,0 +1,297 @@
+            _unityConvertersShow = new AnimBool(_unityConverters.isExpanded);
+            _jsonNetConvertersShow = new AnimBool(_jsonNetConverters.isExpanded);
+
+            _outsideConvertersShow.valueChanged.AddListener(Repaint);
+            _unityConvertersShow.valueChanged.AddListener(Repaint);
+            _jsonNetConvertersShow.valueChanged.AddListener(Repaint);
+            _headerStyle = new GUIStyle { fontSize = 20, wordWrap = true };
+            _boldHeaderStyle = new GUIStyle { fontSize = 20, fontStyle = FontStyle.Bold, wordWrap = true };
+
+            serializedObject.Update();
+            AddAndSetupConverters(_outsideConverters, _outsideConverterTypes, _useAllOutsideConverters.boolValue);
+            AddAndSetupConverters(_unityConverters, _unityConverterTypes, _useAllUnityConverters.boolValue);
+            AddAndSetupConverters(_jsonNetConverters, _jsonNetConverterTypes, _useAllJsonNetConverters.boolValue);
+            serializedObject.ApplyModifiedProperties();
+        }
+
+        public override void OnInspectorGUI()
+        {
+            _isDirty = false;
+            EditorGUILayout.LabelField("Settings for the converters of", _headerStyle);
+            EditorGUILayout.LabelField("Newtonsoft.Json-for-Unity.Converters", _boldHeaderStyle);
+
+            serializedObject.Update();
+
+            EditorGUILayout.Space();
+
+            ToggleLeft(_useUnityContractResolver, "Custom 'Newtonsoft.Json.Serialization.IContractResolver' defined to" +
+                " properly handle the 'UnityEngine.SerializeFieldAttribute' attribute and correctly creates" +
+                " 'UnityEngine.ScriptableObject' via 'ScriptableObject.Create()' instead of the default" +
+                " 'new ScriptableObject()'.");
+
+            EditorGUILayout.Space();
+
+            FoldedConverters(_useAllOutsideConverters, _outsideConverters, _outsideConvertersShow,
+                "Registers all classes outside of the 'Newtonsoft.Json.*' namespace" +
+                " that derive from 'Newtonsoft.Json.JsonConverter' that has a default constructor.");
+
+            EditorGUILayout.Space();
+
+            FoldedConverters(_useAllUnityConverters, _unityConverters, _unityConvertersShow,
+                "Registers all classes inside of the 'Newtonsoft.Json.UnityConverters.*' namespace.");
+
+            EditorGUILayout.Space();
+
+            FoldedConverters(_useAllJsonNetConverters, _jsonNetConverters, _jsonNetConvertersShow,
+                "Registers all classes inside of the 'Newtonsoft.Json.UnityConverters.*' namespace.");
+
+            EditorGUILayout.Space();
+
+            serializedObject.ApplyModifiedProperties();
+
+            if (_isDirty)
+            {
+                UnityConverterInitializer.RefreshSettingsFromConfig();
+            }
+        }
+
+        private void AddAndSetupConverters(SerializedProperty arrayProperty, IList<Type> converterTypes, bool newAreEnabledByDefault)
```

Comment:
Codacy found an issue: [Make 'AddAndSetupConverters' a static method.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5865055)